packages/core/src/domain/configuration/configuration.ts

```diff
@@ -125,11 +130,18 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
     )
   }
 
+  // Build Session Store options
+  const sessionStoreOptions: SessionStoreOptions = {
+    cookie: buildCookieOptions(initConfiguration),
+    allowFallbackToLocalStorage: !!initConfiguration.allowFallbackToLocalStorage,
+  }
+
   return assign(
     {
       beforeSend:
         initConfiguration.beforeSend && catchUserErrors(initConfiguration.beforeSend, 'beforeSend threw an error:'),
-      cookieOptions: buildCookieOptions(initConfiguration),
+      sessionStoreOptions,
+      sessionStoreStrategyType: getSessionStoreStrategyType(sessionStoreOptions),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about having an action more expressive than `get`?
Maybe `selectSessionStoreStrategyType`?

Reply:
👍